### DISCUSSION TITLE
Value of Vehicle Cross-Section

### DISCUSSION THREAD

**Discussion by AbdullahK047** [microsoft/AirSim#3941] [Support Q&A]
Hi,

According to the [AirSim paper](https://microsoft.github.io/AirSim/paper/main.pdf), the drag force is calculated as:

<img src="https://render.githubusercontent.com/render/math?math=|\mathbf{F_d}| = \frac{1}{2}\rho|\mathbf{v}|^2 C_{lin}A">

I have been able to track down values of all the constants from the code except _A_:

- <img src="https://render.githubusercontent.com/render/math?math=\rho = 1.225"> 
- <img src="https://render.githubusercontent.com/render/math?math=C_{lin} = 0.325"> 

So, what is the value of the constant _A_ being used here?

Thanks in advance!

**Comment by jonyMarino** [comment]
Hi @AbdullahK047! what are you trying to accomplish?

**Reply by AbdullahK047** [reply]
Hi! I'm intending to use AirSim for research on Control Barrier Functions at the UPenn [GRASP Lab](https://www.grasp.upenn.edu/), which requires me to write the quadrotor dynamics in control affine form. So the value of the vehicle cross-section is one of the things I couldn't get from the paper and code, and would love to know.

**Reply by jonyMarino** [reply]
Very good! You can also connect Gazebo o JSBSim to AirSim to get different behavior for dynamics. Is that suitable for you?
